### PR TITLE
Added configureBatchActions method to AbstractAdmin

### PR DESCRIPTION
**Pull request by core23**
Added new method to configure batch actions inside the admin class.

Refs https://github.com/sonata-project/SonataAdminBundle/pull/3844

### Changelog

```markdown
### Added
- Added `AbstractAdmin::configureBatchActions` method
```

### To do

- [x] Update the tests
- [X] Update the documentation
- [X] Add an upgrade note

### CODE REVIEW

**Comment by Soullivaneuh** [comment]
I suppose this is related to #3844?

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,8 @@ All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [3.x]
+### Added
```
Comment (Soullivaneuh) [inline CHANGELOG.md]: No need to update the changelog file anymore. :wink:

File: Tests/Fixtures/Admin/ModelAdmin.php [hunk 2]
```diff
@@ -64,4 +64,19 @@ protected function configureShowFields(ShowMapper $showMapper)
...
     }
+
+    /**
+     * @param array $actions
+     *
+     * @return array
+     */
+    protected function configureBatchActions($actions)
+    {
+        $actions['foo'] = array(
```
Comment (Soullivaneuh) [inline Tests/Fixtures/Admin/ModelAdmin.php]: You add a `foo` action but where this is verified?

Comment (core23) [inline Tests/Fixtures/Admin/ModelAdmin.php]: This is still WIP, because tests are missing. I'm working on it ;)

File: Admin/AbstractAdmin.php [hunk 3]
```diff
@@ -2876,6 +2878,18 @@ protected function configureShowFields(ShowMapper $show)
     protected function configureRoutes(RouteCollection $collection)
     {
     }
+    
+    /**
```
Comment (Soullivaneuh) [inline Admin/AbstractAdmin.php]: Wrong indentation.

**Pull request merged**